### DISCUSSION TITLE
Current State of Contributing and Bug Fixes

### DISCUSSION THREAD

**Discussion by AngoMangoOSS** [microsoft/AdaptiveCards#8618] [Q&A]
In your [contributing guidelines](https://github.com/microsoft/AdaptiveCards/blob/dd9627ac96ffb8a4514972947df8b7a822432624/.github/CONTRIBUTING.md#instructions-for-contributing-code) it is said that in order to begin working on a specific bug, that it must be "approved by the Adaptive Cards team". 

What would constitute an approval from the Adaptive Cards team? Are outsiders free to contribute to bugs that have not had any sort of activity (i.e., response from the Adaptive Cards team) on them?

There a few bugs in the issue tracking section that I'd like to check out, however I want to make sure I'm not overstepping my boundaries.

Thank you!

**Comment by jwoo-msft** [comment]
Could you please share the list of bugs to which you would like to contribute? Our team will review these bugs and let you know which ones would generate the most traction. If our interests coincide, we can fast-track the process. Your PR will receive a faster response if you can demonstrate your fix's effectiveness by including video, images, and unit tests, in addition to adhering to good coding practices.
c.c @anna-dingler

**Reply by AngoMangoOSS** [reply]
I'm looking to generally contribute to any amount of bugs that come by, however if you're looking for a specific list, I would look towards tackling:
https://github.com/microsoft/AdaptiveCards/issues/8626
https://github.com/microsoft/AdaptiveCards/issues/8614
https://github.com/microsoft/AdaptiveCards/issues/8603
https://github.com/microsoft/AdaptiveCards/issues/8625
https://github.com/microsoft/AdaptiveCards/issues/8544

There are other bugs that interest me, going farther back, so these are just the start.

I understand that for feature requests, there is a formal submission would have to go through, per your documentation, however is the same true for bugs? Should I take a more simpler approach and request to be assigned to a certain bug, so that I may begin working on it? (That is, if the bug is true).
